shared/constants/types/chat2/index.js

```diff
@@ -24,8 +34,9 @@ export type _State = {
   messageMap: I.Map<Common.ConversationIDKey, I.Map<Message.Ordinal, Message.Message>>, // messages in a thread
   messageOrdinals: I.Map<Common.ConversationIDKey, I.SortedSet<Message.Ordinal>>, // ordered ordinals in a thread
   metaMap: I.Map<Common.ConversationIDKey, Meta.ConversationMeta>, // metadata about a thread, There is a special node for the pending conversation
-  quotingMap: I.Map<Common.ConversationIDKey, QuotedOrdConv>, // current message being quoted
+  quotingMap: I.Map<Common.ConversationIDKey, QuoteInfo>, // current message being quoted
```

Comment:
Is this used anywhere now?

Reply:
Oops, this snuck in during a rebase. Removed.